tests/unit/data/_async/test_client.py

```diff
@@ -245,6 +309,19 @@ async def test__start_background_channel_refresh(self):
             assert ping_and_warm.call_count == 1
             await client.close()
 
+    @CrossSync.pytest
+    async def test__start_background_channel_refresh_disable_refresh(self):
+        client = self._make_client(
+            project="project-id",
+            disable_background_channel_refresh=True,
+        )
+        # should create background tasks for each channel
+        with mock.patch.object(client, "_ping_and_warm_instances", CrossSync.Mock()):
```

Comment:
Do you need to mock this? I think that was done in other tests as a crude way of disabling background refresh